src/tests/multihost/alltests/test_backtrace.py

```diff
@@ -0,0 +1,167 @@
+"""Automation poor man's backtrace
+
+:requirement: Poor Man's Backtrace
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+:bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1949149
+"""
+
+from __future__ import print_function
+import re
+import time
+import pytest
+from sssd.testlib.common.utils import sssdTools
+from sssd.testlib.common.expect import pexpect_ssh
+from constants import ds_instance_name
+
+
+@pytest.mark.usefixtures('setup_sssd', 'create_posix_usersgroups')
+@pytest.mark.backtrace
+@pytest.mark.tier1_2
+class TestPoorManBacktrace(object):
+    """ Check sssd backtrace feature """
+    def test_0001_bz2021196(self, multihost, backupsssdconf):
+        """
+        :title: avoid duplicate backtraces
+        :id: d4d8a0a0-ab90-4c8f-8087-95dc7ad3f3ae
+        :customerscenario: true
+        :bugzilla:
+          https://bugzilla.redhat.com/show_bug.cgi?id=2021196
+          https://bugzilla.redhat.com/show_bug.cgi?id=2021499
+        """
+        hostname = multihost.master[0].sys_hostname
+        bad_ldap_uri = f"ldaps://typo.{hostname}"
+        tools = sssdTools(multihost.client[0])
+        domain_params = {'ldap_uri': bad_ldap_uri}
+        tools.sssd_conf(f'domain/{ds_instance_name}', domain_params)
+        tools.clear_sssd_cache()
+        logfile = '/var/log/sssd/sssd_nss.log'
+        cmd = f'getent passwd fakeuser@{ds_instance_name}'
```

Comment:
What errors will be in `sssd_nss.log` in this case?

Domain will be offline due to failure to connect to "bad_ldap_uri" (and corresponding error/backtrace should be in sssd_$domain.log)
But this probably should not trigger a backtrace in sssd_nss.log...